openedx/adg/lms/applications/constants.py

```diff
@@ -118,3 +118,17 @@
 
 PROGRAM_PREREQ_COURSES_TITLE = f'<strong>{_("Program Prerequisite Courses:")}</strong>'
 BU_PREREQ_COURSES_TITLE = f'<strong>{_("Business Unit Prerequisite Courses:")}</strong>'
+
+
+APPLICATION_WAITLISTED_CONGRATS = _(
+    "Hey {username}, Thank you for visiting again. Your application is still under review."
+)
+APPLICATION_WAITLISTED_INSTRUCTION = f'{_("Admin has waitlisted your application. Please come back and ")}' \
+                                     f'{_("revisit again in sometime to check your application status.")}'
+
+APPLICATION_ACCEPTED_CONGRATS = _(
+    "Congratulations {username}! Your application has been reviewed and accepted by the admin."
+)
+APPLICATION_ACCEPTED_INSTRUCTION = f'{_("Admin will connect with you shortly to begin the process of establishing ")}' \
+                                   f'{_("your business. This will involve legal ")}' \
+                                   f'{_("documentation and on-the-job training and additional details.")}'
```

Comment:
Same here avoid backslashes and use translation method `_()` only once for whole string.